scripts/ci/ci_prepare_packages.sh

```diff
@@ -63,46 +93,12 @@ do
     python3 setup_backport_packages.py "${BACKPORT_PACKAGE}" sdist bdist_wheel >/dev/null
 done
 
-if [[ ${BUILD_COMMON_PROVIDERS_PACKAGE} == "true" ]]; then
-    echo
-    echo "-----------------------------------------------------------------------------------"
-    echo " Preparing backporting package providers (everything)"
-    echo "-----------------------------------------------------------------------------------"
-    echo
-    python3 setup_backport_packages.py providers clean --all
-    python3 setup_backport_packages.py providers sdist bdist_wheel >/dev/null
-fi
+DUMP_FILE="/tmp/airflow_provider_packages_$(date +"%Y%m%d-%H%M%S").tar.gz"
 
-popd || exit 1
+tar -cvzf "${DUMP_FILE}" "dist"
 
-cd "${MY_DIR}/../../" || exit 1
+echo "Packages are in dist bit also tar-gzipped in ${DUMP_FILE}"
 
-if [[ ${BUILD_AIRFLOW_PACKAGE} == "true" ]]; then
-    echo
-    echo "-----------------------------------------------------------------------------------"
-    echo " Preparing apache-airflow package"
-    echo "-----------------------------------------------------------------------------------"
-    echo
-    python3 setup.py clean --all
-    python3 setup.py sdist bdist_wheel >/dev/null
-    echo
-    echo "-----------------------------------------------------------------------------------"
-    echo " Preparing apache-airflow-pinned package"
-    echo "-----------------------------------------------------------------------------------"
-    echo
-    python3 setup.py clean --all
-    python3 setup.py pinned sdist bdist_wheel >/dev/null
-fi
```

Comment:
Where is now the logic for building the package?

Reply:
Good eye. Bad merge.